dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx

```diff
@@ -172,16 +145,33 @@ type ThreeTierChoiceCardsProps = {
 	selectedAmount: number;
 	setSelectedAmount: Dispatch<SetStateAction<number>>;
 	countryCode?: string;
+	variantOfChoiceCard: string;
+};
+
+export const getChoiceCardData = (variant: string) => {
+	switch (variant) {
+		case 'THREE_TIER_CHOICE_CARDS':
+			return ChoiceCardTestData_REGULAR;
+		case 'V1_THREE_TIER_CHOICE_CARDS':
+			return ChoiceCardTestData_V1;
+		case 'V2_THREE_TIER_CHOICE_CARDS':
+			return ChoiceCardTestData_V2;
+		default:
+			return ChoiceCardTestData_REGULAR;
+	}
 };
 
 export const ThreeTierChoiceCards = ({
 	countryCode,
 	selectedAmount,
 	setSelectedAmount,
+	variantOfChoiceCard,
 }: ThreeTierChoiceCardsProps) => {
 	const currencySymbol = getLocalCurrencySymbol(countryCode);
 	const countryGroupId = countryCodeToCountryGroupId(countryCode);
 
+	const Choices = getChoiceCardData(variantOfChoiceCard) as ChoiceInfo[];
```

Comment:
You can remove the `as` here and add the types to the `getChoiceCardData` function as its return type